src/App.js

```diff
@@ -2,59 +2,93 @@
 import React, { useEffect, useState } from 'react';
 import TodoList from './TodoList'
 import AddTodoForm from './AddTodoForm'
+import TodoListItem from './TodoListItem';
 
 
 function App() {
 
   const [todoList, setTodoList] = useState ([])
   const [isLoading, setIsLoading] = useState (true)
+  const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`
 
-  useEffect(() => {
-    new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve(
-          {data: {
-          todoList: JSON.parse(localStorage.getItem("savedTodoList")) || []
-        }}
-        )
-      }, 2000)
-    })
-    .then((result) => {
-      setTodoList(result.data.todoList)
-      setIsLoading(false)
+  const fetchData = async () => {
+    const options = {
+      method: 'GET',
+      headers: {
+        Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_KEY}`,
+      }
     }
-    )
-  }, [])
 
-  useEffect (() => {
-    if (!isLoading)  {
-      localStorage.setItem("savedTodoList", JSON.stringify(todoList))
+    try {
+      const response = await fetch (url, options)
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`)
+      }
+      const data = await response.json()
+      const todos = data.records.map((todo) => {
+        return {
+          title: todo.fields.title,
+          id: todo.id
+        }
+
+      })
+      setTodoList(todos)
+      setIsLoading(false)
+
+    } catch (error) {
+      console.log(error)
     }
```

Comment:
💯 Wow!! Super impressive here. Looks like professional code here with the `map` function :)